charts/prometheus-cloudwatch-exporter/templates/deployment.yaml

```diff
@@ -26,6 +26,7 @@ spec:
       {{- with .Values.priorityClassName }}
       priorityClassName: "{{ . }}"
       {{- end }}
+      imagePullSecrets: {{ .Values.image.pullSecrets }}
```

Comment:
Should we add an if statement around this to only render it when not empty?

Reply:
Sure - fixed by doing what is done in the prometheus chart.